docs/en/testnet-plasma.md

```diff
@@ -1,36 +1,35 @@
 ---
-id: testnet-plasma
-title: PlasmaChain Extdev
-sidebar_label: PlasmaChain Extdev
+id: loom-testnet
+title: Extdev Testnet
+sidebar_label: Extdev Testnet
 ---
 
-This page provides information on where and how to access the Extdev network.
-If you want more details, see our [guide](deploy-loom-testnet.html) on deploying to Loom testnet.
+This page provides information on where and how to access the Extdev Testnet network. If you want more details, see our [guide](deploy-loom-testnet.html) on deploying to Extdev Testnet.
```

Comment:
`Tesnet network` is redundant just `Tesnet` will do.

Reply:
Fixed.